snapcraft.yaml

```diff
@@ -29,6 +30,9 @@ description: |
        - read/write to gsettings:
          plugs: [gsettings, home]
          (note that the home plug is needed to read new value)
+       - use of the shared platform snap content:
+         plugs: [ubuntu-app-platform]
+         (content interface plug to use ubuntu-app-platform slot needed)
```

Comment:
oh, as well, please provide default-provider: ubuntu-app-platform (snap name). Once this is implemented, installing the application will install the content interface snap at the same time (it's not implemented though right now)